qiskit_experiments/library/__init__.py

```diff
@@ -0,0 +1,78 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+======================================================
+Experiment Library (:mod:`qiskit_experiments.library`)
+======================================================
+
+.. currentmodule:: qiskit_experiments.library
+
+Verification Experiments
+========================
+
+.. autosummary::
+    :toctree: ../stubs/
+
+    StandardRB
+    InterleavedRB
+    StateTomography
+    ProcessTomography
+    QuantumVolume
+
+Characterization Experiments
+============================
+
+.. autosummary::
+    :toctree: ../stubs/
+
+    T1
+    T2Ramsey
+    QubitSpectroscopy
+    EFSpectroscopy
+
+Calibration Experiments
+=======================
+
+.. autosummary::
+    :toctree: ../stubs/
+
+    DragCal
+    Rabi
+    EFRabi
+    QubitSpectroscopy
```

Comment:
spectroscopy belongs to both characterization and calibration?

Reply:
I'm not sure, I was guessing from the calibration module doc string. I was thinking that any experiment that you can use with the calibration management stuff should also be listed under calibration to identity. If these shouldn't be here I can remove them (thoughts @eggerdj ?)

Reply:
There was discussion regarding this. Actually spectroscopy is an experiment that determines qubit frequency (=characterization), however, we can use this to determine the drive frequency (=calibration) because we drive the stimulus at frequency of qubit. This means user can still run spectroscopy without updating any parameter.

In this case, we can classify such experiment into characterization, as @blakejohnson recommended. I'm fine with placing the code in the characterization but also adding symbolic link from the calibration.

Reply:
For those cases I was proposing duplicating it in both lists here

Reply:
For now ive removed it from calibration list, if we decide we want to readd them here that can be done later